hpc-code

```diff
@@ -0,0 +1,82 @@
+## Running previous code
+
+import pandas as pd
+from sklearn.ensemble import RandomForestRegressor
+from sklearn.metrics import mean_absolute_error
+from sklearn.model_selection import train_test_split
+from sklearn.tree import DecisionTreeRegressor
+
+# Set up code checking
+import os
+if not os.path.exists("../input/train.csv"):
```

Comment:
Relative paths make it easier to deploy the code, because you can just "zip" this file up and people can run it!